ports/psoc6/mpconfigport.h

```diff
@@ -116,23 +108,16 @@
 #define MICROPY_PY_MACHINE_SOFTSPI              (0)
 #define MICROPY_PY_ONEWIRE                      (0)
 #define MICROPY_VFS                             (1)
-#define MICROPY_VFS_LFS2                        (1)
+
+// MICROPY_VFS_LFS2 set in mpconfigport.mk
```

Comment:
If this is done in the mpconfigport.mk, why not the FAT? or any other which is in included based on preprocessor guards? 
Same for any others which are enabled from makefile side, and the adding the corresponding flags -> https://github.com/jaenrig-ifx/micropython/blob/ports/psoc6/extmod/extmod.mk#L75